Content/NPCs/Moonstone/DreambeastProjectiles.cs

```diff
@@ -0,0 +1,258 @@
+﻿using StarlightRiver.Helpers;
+using System;
+using System.Collections.Generic;
+using Terraria.DataStructures;
+using Terraria.Graphics.Effects;
+using Terraria.ID;
+
+namespace StarlightRiver.Content.NPCs.Moonstone
+{
+	public class DreambeastProj : ModProjectile, IDrawAdditive
+	{
+		public override string Texture => AssetDirectory.MoonstoneNPC + Name;
+
+		public override void SetDefaults()
+		{
+			Projectile.hostile = true;
+			Projectile.width = 36;
+			Projectile.height = 20;
+			Projectile.penetrate = 1;
+			Projectile.timeLeft = 45;
+			Projectile.tileCollide = false;
+			Projectile.ignoreWater = true;
+			Projectile.damage = 66;
+		}
+
+		public override void SetStaticDefaults()
```

Comment:
Bit of a nit but in alot of other classes we have SetStatic before SetDefaults, might be worth to change it around for consistency